### PR TITLE
chore: upgrade Play to v3.0.1

### PR DESCRIPTION
**Pull request by tjsilver**
## What is the purpose of this change?

This upgrades Play to v3. SBT has also been upgraded to latest.

## What is the value of this change and how do we measure success?

Play 3 uses Pekko, so upgrading removes our reliance on Akka.

### CODE REVIEW

**Review by adamnfish — APPROVED** [review]
LGTM, if i9t works! (aside from possibly being less specific for asdf)

**Review by AshCorr — APPROVED** [review]
Looks good!

I think [this comment](https://github.com/guardian/janus-app/pull/394/files#diff-8fd74420ca55aa91d4c53775d2a55a999c405b5a4fafeaf73b7f0ef83e1f3b0cL9) no longer applies as Play 2 was probably the blocker!

File: .tool-versions [hunk 1]
```diff
@@ -0,0 +1 @@
+java corretto-11.0.15.9.1
```
[the remark this answers is not in the document]
Comment (tjsilver) [reply .tool-versions:1]: Good spot - didn't meant to commit this!

**Comment by tjsilver** [comment]
> Looks good!
> 
> I think [this comment](https://github.com/guardian/janus-app/pull/394/files#diff-8fd74420ca55aa91d4c53775d2a55a999c405b5a4fafeaf73b7f0ef83e1f3b0cL9) no longer applies as Play 2 was probably the blocker!

Good spot - I've removed both the comment and the workaround which seems to be [redundant](https://github.com/sbt/sbt/pull/7021) anyway.

**Pull request merged**